src/backend/metal/src/native.rs

```diff
@@ -55,8 +55,8 @@ unsafe impl Sync for ShaderModule {}
 #[derive(Clone, Debug, Default, Hash, PartialEq, Eq)]
 pub struct RenderPassKey {
     // enough room for 4 color targets + depth/stencil
```

Comment:
actually, still correct. The old comment was a bit unclear: I assumed that the user doesn't need to clear all 4 color surfaces (since GTA doesn't do that), but on Vulkan it's fairly cheap, so I assume people don't bother saving on that. The new code is more strict: 4 color operations, 1 depth, 1 stencil, 4 data values per color. This results in 6 operations and 18 clear words, which is rounded up to 20 because of `SmallVec` limitations.